src/main/asciidoc/project/lra/jaxrs.adoc

```diff
@@ -5,3 +5,13 @@ Primary support is for JAX-RS based services to participate in transactional Lon
 Full details are available in the MP-LRA specification document https://github.com/eclipse/microprofile-lra/blob/master/spec/src/main/asciidoc/microprofile-lra-spec.adoc.
 Also look in the xref:examples.adoc#lra_examples[examples section] below and in the MP-LRA TCK.
 
+== Configuring the location of the coordinator when starting new Long Running Actions
+
+Services should be configured with a comma separated list of URLs of coordinators using a property called `lra.coordinator.urls`, and if unset then `http://localhost:8080/lra-coordinator` will be used. When a service starts a new LRA one of the URLs will be chosen in a "round robin" fashion. Note that the chosen coordinator must be available for the lifetime of the LRA (or restarted on the same endpoint, with access to the same object store, if it fails). The https://smallrye.io/smallrye-stork/latest[Smallry Stork service discovery and client-side load-balancing framework] is used to support a static list of coordinators.
+
+Future revisions of LRA will support the configuration of:
+
+* other load balancing algorithms when choosing a coordinator to start a new LRA
+* other service discovery backends, besides the simple static list defined by the value of `lra.coordinator.urls`.
+
+Please refer to the https://smallrye.io/smallrye-stork/latest[Load-Balancing] and https://smallrye.io/smallrye-stork/latest[Smallrye Stork Service Discovery] sections in the documentation for a list of supported algorithms (currently Round Robin, Random, Least Requests, Least Response Time, Power Of Two Choices, and Sticky) and frameworks (which currently includes Consul, DNS, Kubernetes, KNative, Eureka, Composite and Static List).
```

Comment:
As the lists of `currently available...` might become outdated soon or later, I was wondering if we can remove them.